Documentation/writing-custom-queries.md

```diff
@@ -3,10 +3,11 @@
 One of the main goals of Operator Metering is to be flexible, and extensible.
 The way this has been done is to use Kubernetes Custom Resources as a way of letting users add to, or expand upon the built-in reports and metrics that the operator already has.
 
-The primary Custom Resources that allow this are the [ReportPrometheusQuery][reportprometheusqueries], [ReportDataSource][reportdatasources], and the [ReportGenerationQuery][reportgenerationqueries].
+The primary Custom Resources that allow this are the ReportDataSource][reportdatasources], and the [ReportGenerationQuery][reportgenerationqueries].
```

Comment:
Missing the first open bracket in `ReportDataSource]`

Reply:
Fixed